scripts/stac/imagery/collection.py

```diff
@@ -39,8 +41,8 @@
 from scripts.stac.util.media_type import StacMediaType
 from scripts.stac.util.stac_extensions import StacExtensions
 
-ANY_ORTHO_AERIAL_PHOTOS = {AERIAL_PHOTOS, URBAN_AERIAL_PHOTOS, RURAL_AERIAL_PHOTOS}
-IMAGERY = {SCANNED_AERIAL_PHOTOS, SATELLITE_IMAGERY, *ANY_ORTHO_AERIAL_PHOTOS}
+ANY_ORTHO_AERIAL_PHOTOS = {AERIAL_PHOTOS, URBAN_AERIAL_PHOTOS, RURAL_AERIAL_PHOTOS, NEAR_INFRARED_AERIAL_PHOTOS}
+IMAGERY = {SCANNED_AERIAL_PHOTOS, SATELLITE_IMAGERY, NEAR_INFRARED_SATELLITE_IMAGERY, *ANY_ORTHO_AERIAL_PHOTOS}
```

Comment:
NIT: could group `SATELLITE_IMAGERY` and `NEAR_INFRARED_SATELLITE_IMAGERY` in the same fashion as `ANY_ORTHO_AERIAL_PHOTOS`